homeassistant/bootstrap.py

```diff
@@ -86,14 +86,6 @@ def async_from_config_dict(config: Dict[str, Any],
     if enable_log:
         async_enable_logging(hass, verbose, log_rotate_days, log_file)
 
-    if sys.version_info[:2] < (3, 5):
-        _LOGGER.warning(
```

Comment:
Might be important to keep a check...

change **warning** to **error** and abort execution